e2e-test/cypress/integration/project_edit_spec.js

```diff
@@ -0,0 +1,60 @@
+describe("Project Edit", function() {
+  let projects = undefined;
+
+  before(() => {
+    cy.login();
+    cy.fetchProjects().then(p => (projects = p));
```

Comment:
this requires projects to be added. to make it independent of provisioning we should create the project for testing here

Reply:
Done. Since most tests are done with user "mstein" should we remove all view permissions on projects created in e2e-test phase?

Reply:
no, since the tests are usually run in a separate container in the CI/CD pipeline this is not a very big issue. users testing locally can perform this action manually if they like